pkgs/tools/package-management/nix-prefetch-scripts/default.nix

```diff
@@ -66,6 +53,9 @@ let
     };
 in
 rec {
+  # No explicit dependency on Nix, as these can be used inside builders,
+  # and thus will cause dependency loops. When used _outside_ builders,
+  # we expect people to have a Nix implementation available ambiently.
```

Comment:
Only `nix-prefetch-git` can be used.
Though we can keep the comment in this place, it's fine.